### PR TITLE
Support ARM64 architecture

### PR DESCRIPTION
**Pull request by Alexey-Ayupov**
# Description
In this PR 
Added: 
 - Support macOS 13 clean image generation on the arm64 platform 

Fixed: 
 - macOS 12 tried to download the major update to macOS Ventura

#### Related issue:
https://github.com/actions/runner-images-internal/issues/4941

## Check list
- [x] Related issue / work item is attached
- [ ] Tests are written (if applicable)
- [ ] Documentation is updated (if applicable)
- [ ] Changes are tested and related VM images are successfully generated

### CODE REVIEW

File: images.CI/macos/anka/Service.Helpers.psm1 [hunk 1]
```diff
@@ -37,6 +37,84 @@ function Get-AvailableVersions {
...
+
+
+    Write-Host "`t[*] Finding available full installers"
+    if ($DownloadLatestVersion -eq $true) {
+        $targetVersion = Get-AvailableIPSWVersions -IsLatest $true -MacOSCodeNameOrVersion $MacOSName
+        Write-host "`t[*] The 'DownloadLatestVersion' flag is set to true. Latest macOS version is '$MacOSName' - '$targetVersion' now"
+    } elseif ($BetaSearch -eq $true) {
+        $targetVersion = Get-AvailableIPSWVersions -IsBeta $true -MacOSCodeNameOrVersion $MacOSName
+        Write-host "`t[*] The 'BetaSearch' flag is set to true. Latestbeta macOS version is '$MacOSName' - '$targetVersion' now"
+    }else {
```
Comment (mikhailkoliada) [inline images.CI/macos/anka/Service.Helpers.psm1:93]: add space

Comment (Alexey-Ayupov) [reply images.CI/macos/anka/Service.Helpers.psm1:93]: Fixed

File: images.CI/macos/anka/Service.Helpers.psm1 [hunk 2]
```diff
@@ -150,11 +228,22 @@ function Install-SoftwareUpdate {
...
-        [string] $HostName
+        [string] $HostName,
+        [array] $listOfUpdates
     )
-
-    $command = "sudo /usr/sbin/softwareupdate --all --install --restart --verbose"
-    Invoke-SSHPassCommand -HostName $HostName -Command $command
+    $osVersion = [Environment]::OSVersion
+    $osVersionMajorMinor = $osVersion.Version.ToString(2)
+    if ($osVersion.Version.Major -eq "12") {
```
Comment (mikhailkoliada) [inline images.CI/macos/anka/Service.Helpers.psm1:236]: could you please left a comment why we do so

Comment (Alexey-Ayupov) [reply images.CI/macos/anka/Service.Helpers.psm1:237]: Done

**Review by mikhailkoliada — APPROVED** [review]

**Pull request merged**